TurkishFilter/sections/specific.txt

```diff
@@ -42,6 +42,9 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
 sgkbilgisi.com##.fdsphvn
+61saat.com#$##modal-ads { display: none !important; }
+61saat.com#$#.modal-backdrop { display: none !important; }
+61saat.com#$#body { overflow: visible !important; padding-right: 0 !important; }
```

Comment:
`padding-right: 0 !important;` is not required

Reply:
@Alex-302 On my end, the padding changes.

Reply:
I see no difference. Do you see horyzontal scroll without it or what?

Reply:
Nope. It's just normal. @zloyden Can explain better. 
![image](https://user-images.githubusercontent.com/76908410/113862842-82061200-97db-11eb-80c8-b21470cbfbba.png)

Reply:
Now I see)